src/components/subgoal/Subgoal-Assignment-Modal.tsx

```diff
@@ -14,15 +12,21 @@ import {
   ListItemText,
 } from "@mui/material";
 import { useState } from "react";
+import $subgoal from "./Subgoal-Assignment-Modal.module.css";
 
 interface SubgoalAssignmentModalProps {
   isOpen: boolean;
   onClose: () => void;
+  subgoal_id: string;
 }
 
 export const SubgoalAssignmentModal = (props: SubgoalAssignmentModalProps) => {
   const [selectedParaIds, setSelectedParaIds] = useState<string[]>([]);
+  const [currentModalSelection, setCurrentModalSelection] = useState(1);
```

Comment:
🐧 maybe something like

```suggestion
  const [currentModalSelection, setCurrentModalSelection] = useState<"PARA_SELECTION" | "SCHEDULING">("PARA_SELECTION");
```

to make this a little more clear